src/coreclr/src/inc/pedecoder.h

```diff
@@ -131,13 +131,15 @@ class PEDecoder
     PEDecoder();
     PEDecoder(void *flatBase, COUNT_T size);              // flatBase is the raw disk layout data (using MapViewOfFile)
     PEDecoder(PTR_VOID mappedBase, bool relocated = FALSE);  // mappedBase is the mapped/expanded file (using LoadLibrary)
+    virtual ~PEDecoder();
 
     void Init(void *flatBase, COUNT_T size);
     HRESULT Init(void *mappedBase, bool relocated = FALSE);
     void   Reset();  //make sure you don't have a thread race
 
     PTR_VOID GetBase() const;            // Currently loaded base, as opposed to GetPreferredBase()
     BOOL IsMapped() const;
+    virtual BOOL IsInBundle() const;
```

Comment:
I do not think it is worth introducing virtual methods into PEDecoder just for debug-only check. Instead, I would delete the check that needs this.

Reply:
(The check is not strictly debug-only, but it should be fine to delete it.)

Reply:
@jkotas - do you mean - just relax the alignment requirement to 16 byte unconditionally?

Reply:
Yes. Or even delete `CHECK(CheckAligned(m_base, GetOsPageSize()));` without replacement.